### PR TITLE
Pomegranate Compatibility: Frozenset -> Tuple

### PR DESCRIPTION
**Pull request by nikisix**
Found an issue with the bayes net structure that arose from a change in [#22](https://github.com/sidewalklabs/doppelganger/pull/22). Reverting back to tuples for compatibility with pomegranate.

---
This change is [Reviewable](https://reviewable.io/reviews/sidewalklabs/doppelganger/31)

### CODE REVIEW

**Comment by katbusch** [comment]
Thanks @nikisix!
This also fixes #30, right?

Can you investigate whether we can cover this bug in our tests so that we don't accidentally reintroduce it?

File: test/test_bayes_net.py [hunk 1]
```diff
@@ -186,13 +195,36 @@ def test_generate_household(self):
...
+        self.assertEquals(
+            type(
+                BayesianNetworkModel.train(
+                    training_data,
+                    person_structure,
+                    self._person_fields,
+                    prior_data=None
+                )
+            ),
+            bayesnets.BayesianNetworkModel
```
Comment (nikisix) [inline test/test_bayes_net.py]: Wasn't sure about exactly which logical condition would be best to check for. Important thing is that the train call to pomegranate doesn't crash.

**Comment by katbusch** [comment]
Awesome, thanks for fixing this and for writing the test! I've requested a couple small changes as cleanup!

---

Review status: 0 of 4 files reviewed at latest revision, 4 unresolved discussions, some commit checks failed.

---

*[README.md, line 23 at r2](https://reviewable.io:443/reviews/sidewalklabs/doppelganger/31#-KrYO-F8TnLGmLme-mFq:-KrYO-F9CAefEcSL8fd7:bb752ax) ([raw file](https://github.com/sidewalklabs/doppelganger/blob/05f1a70c2561c98565cdeb93a684db04acc14a3a/README.md#L23)):*
> ```Markdown
> pip install .
> ```
> Or for developmers
> ```

Typo

---

*[test/test_bayes_net.py, line 202 at r2](https://reviewable.io:443/reviews/sidewalklabs/doppelganger/31#-KrYNF3l_Mv2_T9iFHR4:-KrYNF3l_Mv2_T9iFHR5:b-5hlewc) ([raw file](https://github.com/sidewalklabs/doppelganger/blob/05f1a70c2561c98565cdeb93a684db04acc14a3a/test/test_bayes_net.py#L202)):*
> ```Python
>         structure = bayesnets.define_bayes_net_structure(nodes, edges)
>         self.assertSequenceEqual(structure, ((), (0,), (0, 1)))
>         #self.assertSequenceEqual(structure, (frozenset([]), frozenset([0]), frozenset([0, 1])))
> ```

Can you delete this commented out line?

---

*[test/test_bayes_net.py, line 215 at r2](https://reviewable.io:443/reviews/sidewalklabs/doppelganger/31#-KrYND-ocO5vdGDj2fU9:-KrYND-ocO5vdGDj2fUA:b9ruhmp) ([raw file](https://github.com/sidewalklabs/doppelganger/blob/05f1a70c2561c98565cdeb93a684db04acc14a3a/test/test_bayes_net.py#L215)):*
> ```Python
>             person_training_data, self._person_fields(), segmenter=self._person_segmenter()
>         )
>         
> ```

Can you delete this trailing whitespace?

---

*[test/test_bayes_net.py, line 225 at r2](https://reviewable.io:443/reviews/sidewalklabs/doppelganger/31#-KrWu3yodlpAvPz1VuNA-r2-225:-KrYNfHM85pQZtwJoPSS:b2m8x31) ([raw file](https://github.com/sidewalklabs/doppelganger/blob/05f1a70c2561c98565cdeb93a684db04acc14a3a/test/test_bayes_net.py#L225)):*
<details><summary><i>Previously, nikisix wrote…</i></summary><blockquote>

Wasn't sure about exactly which logical condition would be best to check for. Important thing is that the train call to pomegranate doesn't crash.
</blockquote></details>

Looks good! It is certainly unusual but your comment explains why

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/sidewalklabs/doppelganger/31#-:-KrYO1xXYPuQGgSkVsvy:bs4od56)*

**Comment by katbusch** [comment]
Looks like the linter is still mad:
> ./test/test_bayes_net.py:204:63: W291 trailing whitespace
./test/test_bayes_net.py:227:5: E303 too many blank lines (2)

Should just be `flake8` to run it locally

---

Review status: 0 of 4 files reviewed at latest revision, 4 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/sidewalklabs/doppelganger/31#-:-KrY_ofL0lUKevLznjoj:bavuig)*

**Comment by katbusch** [comment]
:lgtm:

Thanks @nikisix!

---

Review status: 0 of 4 files reviewed at latest revision, 4 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/sidewalklabs/doppelganger/31#-:-KrbBAVYyK--19tDl8qu:bd06iuq)*

**Pull request merged**